packages/@uppy/image-editor/src/Editor.jsx

```diff
@@ -140,7 +140,11 @@ export default class Editor extends Component {
           className="uppy-u-reset uppy-c-btn"
           onClick={() => {
             this.cropper.reset()
-            this.cropper.setAspectRatio(0)
+            if (opts.cropperOptions.initialAspectRatio) {
```

Comment:
Instead of doing this conditionally here, probably better to set a default here and then we can just pass the option.

https://github.com/transloadit/uppy/blob/a0657db5a02820762865d8836f331f202c544c4e/packages/%40uppy/image-editor/src/ImageEditor.jsx#L19-L27

Reply:
Agreed, let's do

```js
const defaultCropperOptions = {
   //...current stuff,
  initialAspectRatio: 0
}
```

and then do

```js
this.cropper.reset()
this.cropper.setAspectRatio(opts.cropperOptions.initialAspectRatio)
```

Agreed with the change otherwise!